src/core/locator/qgslocator.h

```diff
@@ -153,6 +160,19 @@ class CORE_EXPORT QgsLocator : public QObject
      */
     void foundResult( const QgsLocatorResult &result );
 
+    /**
+     * Emitted when locator has begun a search, before actually preparing it.
+     * \since QGIS 3.16
+     */
+    void searchBegan();
```

Comment:
What is the purpose of this? I don't see it currently used in this PR.

Reply:
I thought it would be required to update the completion text in the line edit but apparently not.
Removed